.github/workflows/code-health.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Black Check
-        uses: jpetrucciani/black-check@20.8b1
+        uses: jpetrucciani/black-check@7f5b2ad20fa5484f1884f07c1937e032ed8cd939
```

Comment:
Q: using a version here directly doesn't work?

Reply:
There's no tag for the latest black release: https://github.com/jpetrucciani/black-check